bin/ansible-connection

```diff
@@ -147,7 +147,6 @@ class ConnectionProcess(object):
 
     def command_timeout(self, signum, frame):
         display.display('command timeout triggered, timeout value is %s secs' % self.connection.get_option('persistent_command_timeout'), log_only=True)
-        self.shutdown()
```

Comment:
This will result in stray socket in case and possibly a infinite wait if the remote host stops responding after a certain command is executed. With https://github.com/ansible/ansible/pull/42847 PR user can now increase command timeout value per task basis. imo increasing command timeout is the right fix instead of handling it in code

Reply:
1. Can you tell exactly why socket will be left stray. ansible-connection has many ways to clear PC ? if there is something left behind we need to fix it .
2. PR #42847 solves only limited use-cases where I know approx command_timeout beforehand
3. It fixes backward compatibility as mentioned in PR #42340
4. IMO, definition of persistent-connection is a socket will be used till the lifetime of playbook. so by forcing it closed it in midway, we are breaking the whole idea of persistency ?